app.rb

```diff
@@ -0,0 +1,61 @@
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative 'lib/peep'
+require_relative 'lib/user'
+require_relative 'lib/database_connection_setup'
+
+class Chitter < Sinatra::Base
+  enable :sessions, :method_override
+  register Sinatra::Flash
+
+  get '/' do
+    @user = User.get session[:user_id]
+    @peeps = Peep.all order: [:id.desc]
```

Comment:
If someone in your team, who is not as up to speed as you are, were to read this, they may be a bit confused as to what this line does. Perhaps commenting on your work would be good practice for teamwork? Alternatively, is there anything you could do to improve the readability of this code?

Reply:
i think maybe this is too logiccy for the controller, and i should have refactored it out into the model, wrapped in a nice method name, which would make this more readable.

Reply:
fixed